lib/Stack2nix/Cache.hs

```diff
@@ -14,14 +15,13 @@ readCache :: FilePath
                  , String -- pkgname
                  , String -- nixexpr-path
                  )]
-readCache f = fmap (toTuple . words) . lines <$> readFile f
+readCache f = fmap (toTuple . words) . lines <$!> readFile f
```

Comment:
Sorry I somehow missed this PR.  I think <$!> here only uses `seq` on the lazy string (not `deepseq`), will that be enough to force the whole file to be read and closed?  In the past I have used [readFile from the strict package](https://hackage.haskell.org/package/strict-0.4.0.1/docs/System-IO-Strict.html#v:readFile) and it uses `seq`, but on the `length` of the string (forcing the file to be read to the end).

Reply:
Oh yes you're right, it would only force the first character to be read.